### DISCUSSION TITLE
How can I put the logo of my project instead of the project name on the top left

### DISCUSSION THREAD

**Discussion by bora-7** [thuliteio/doks#1224] [Q&A]
<img width="1426" alt="Screenshot 2024-04-16 at 00 33 37" src="https://github.com/gethyas/doks/assets/76407294/a168641e-5e35-4f55-9927-96d064919c77">

Instead of the project name I want to put the project png. How can I do that?

**Comment by bora-7** [comment]
Bump

**Reply by h-enk** [reply]
Copy `node_modules/@hyas/doks-core/layouts/partials/header/header.html` to `/layouts/partials/header/header.html` and replace `{{ .Site.Title }}` with your logo — on lines 21 and 85

Here's how I did that for getdoks.org: https://github.com/gethyas/getdoks.org/blob/main/layouts/partials/header/header.html#L20-L25
